src/main/java/org/jenkinsci/plugins/ownership/model/workflow/OwnershipGlobalVariable.java

```diff
@@ -0,0 +1,148 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.jenkinsci.plugins.ownership.model.workflow;
+
+import com.synopsys.arc.jenkins.plugins.ownership.OwnershipDescription;
+import com.synopsys.arc.jenkins.plugins.ownership.jobs.JobOwnerHelper;
+import com.synopsys.arc.jenkins.plugins.ownership.nodes.NodeOwnerHelper;
+import groovy.lang.Binding;
+import hudson.Extension;
+import hudson.model.Node;
+import hudson.model.Run;
+import java.io.IOException;
+import java.io.InputStream;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import jenkins.model.Jenkins;
+import org.apache.commons.io.IOUtils;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.ProxyWhitelist;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.StaticWhitelist;
+import org.jenkinsci.plugins.workflow.cps.CpsScript;
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable;
+import org.jenkinsci.plugins.workflow.support.steps.build.RunWrapper;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+/**
+ * Provides {@link OwnershipDescription}s for different item types.
+ * @author Oleg Nenashev
+ * @since TODO
+ */
+@Extension
+public class OwnershipGlobalVariable extends GlobalVariable {
+
+    @Override 
+    public final Object getValue(CpsScript script) throws Exception {   
+        final Binding binding = script.getBinding();
+        final Object loadedObject;
+        if (binding.hasVariable(getName())) {
+            loadedObject = binding.getVariable(getName());
+        } else {
+            // Note that if this were a method rather than a constructor, we would need to mark it @NonCPS lest it throw CpsCallableInvocation.
+            loadedObject = script.getClass().getClassLoader()
+                    .loadClass(getClassName())
+                    .getConstructor(CpsScript.class).newInstance(script);
+            binding.setVariable(getName(), loadedObject);
+        }
+        return loadedObject;
+    }
```

Comment:
I mostly think about moving https://github.com/jenkinsci/workflow-remote-loader-plugin/blob/master/src/main/java/org/jenkinsci/plugins/workflow/remoteloader/GroovyFileGlobalVariable.java into the Pipeline core plugin. It would also move this code. @jglick WDYT?